src/sc_kpm/sc_server.py

```diff
@@ -73,14 +77,21 @@ def disconnect(self) -> None:
     def add_modules(self, *modules: ScModuleAbstract) -> None:
         if self.is_registered:
             self._register(*modules)
-        self._modules.extend(modules)
+        self._modules |= {*modules}
```

Comment:
Did we have tests for **ScServer** class? I think this operator was included in python 3.10+ versions, so library wouldn't be compatible with previous versions of Python

Reply:
That's set's operator, it was defined long time ago. Also I write on python version 3.8, so there will be no problems with it